app/controllers/groups_controller.rb

```diff
@@ -15,13 +15,18 @@ def assign
     user.group_id = group.id
 
     if user.save
-      redirect_to root_path
+      redirect_to group_path(group)
```

Comment:
We're going to have merge conflicts here. I will merge this pull request and fix it on my end.